tests/utils/dockercli/swarm.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
```

Comment:
As we discussed offline, we should keep each PR small enough so that it’s easier to review and also keep the dependency to the minimum. The current PR is probably a special case as there's only one test case, but it's still pretty big and complicated. Since the review is almost done, I'd prefer to keep the current PR as is instead of wasting time to split it into two.